src/components/Charts/statechart.js

```diff
@@ -74,10 +74,10 @@ function StateChart(props) {
     events: ['mousemove', 'mouseout', 'touchstart', 'touchmove', 'touchend'],
     layout: {
       padding: {
-        left: 20,
-        right: 20,
+        left: 30,
+        right: 30,
```

Comment:
What's the need for this change? @anukaal

Reply:
By changing this padding it will Look more relevant and big in size , it is looking small when using the sites in mobile

Reply:
i want to contact you

Reply:
through my mail which is in my GitHub account

Reply:
Its your anukaal@outlook.com mail?

Reply:
Yes